include/ddc/discrete_domain.hpp

```diff
@@ -418,6 +430,12 @@ struct cartesian_prod<DDom1, DDom2, Tail...>
 template <typename... DDom>
 using cartesian_prod_t = typename cartesian_prod<DDom...>::type;
 
+// Remove dimensions from a domain type
+template <typename DDom, typename... DDim>
+using remove_dims_of_t = ddc::detail::convert_type_seq_to_discrete_domain<ddc::type_seq_remove_t<
+        ddc::detail::convert_discrete_domain_to_type_seq<DDom>,
+        ddc::detail::TypeSeq<DDim...>>>;
+
```

Comment:
I would consider using the function `remove_dims_of` to avoid repeating the logic.
```suggestion
// Remove dimensions from a domain type
template <typename DDom, typename... DDims>
using remove_dims_of_t = decltype(remove_dims_of<DDims...>(std::declval<DDom>()));

```

Or the reverse, use it inside `remove_dims_of`.